lib/logzio-nodejs.js

```diff
@@ -188,10 +188,13 @@ class LogzioLogger {
         }
     }
 
-    log(msg) {
+    log(msg, obj = null) {
         if (this.closed === true) {
             throw new Error('Logging into a logger that has been closed!');
         }
+        if (obj){
```

Comment:
```suggestion
        if ([null, undefined].includes(obj)){
```
To better check if the `obj` is defined